deepchem/models/tests/test_dtnn.py

```diff
@@ -0,0 +1,39 @@
+import os
+import numpy as np
+import pytest
+from scipy import io as scipy_io
+
+from deepchem.data import NumpyDataset
+try:
+    from deepchem.models.torch_models import DTNNModel
+except ModuleNotFoundError:
+    pass
+
+
+@pytest.mark.torch
+def test_dtnn():
+    current_dir = os.path.dirname(os.path.abspath(__file__))
+    input_file = os.path.join(current_dir, "assets/example_DTNN.mat")
+
```

Comment:
We should add an additional test that uses the associated featurizer

Reply:
@rbharath can you tell which featurizer to use in test and example. Me and mentors are having a slight confusion about that.

Reply:
@arunppsg do you know where this data from tests/assets/example_dtnn.mat comes from?

Reply:
@GreatRSingh I think bharath is suggesting you to take out the `compute_features_on_batch` method and make it its own featurizer so the test code is easier to read and the featurization can be used on other models

Reply:
> @arunppsg do you know where this data from tests/assets/example_dtnn.mat comes from?

Not sure. Is the question about which dataset the data is subset of? I think it is a dangling file - not being used in tests or in the codebase.

Reply:
Done